bsip-0056.md

```diff
@@ -0,0 +1,314 @@
+many customers at the same time.
+
+In contrast, decentralized exchanges do not require a trusted third party to hold customer funds in custody and offer
+high transparency through the audit trail of the blockchain and autonomy of order matching as part of the "smart
+contracts" that execute the trading. Most blockchain technologies, however, have the disadvantage of high latency (bitcoin transaction are
+processed only once every 10 minutes on average, BitShares transactions are processed every 3 seconds) and thus do not
+allow for HFT. Additionally, sophisticated trading strategies cannot be applied as they require some degree of
+confidentiality (e.g. the price limit of stop-loss orders).
+
+The concept proposed in this document tries to benefit from both worlds while offering additional features to traders.
+This is done by filling the gap between centralized and decentralized exchanges.
+
+It is proposed to use the blockchain as settlement layer for off-chain trading platforms while removing the default risk
+of the centralized trading platform. While decentralized exchanges follow the concept "the trade is the settlement", we
+here propose to let trades happen outside the blockchain while settlements are performed regularly on-chain. 
+
+## Savings Account
+
+Managed vesting balances furthermore allow to setup *savings accounts* (or rather *savings balances*) which are balances
+that can only be accessed after a certain amount of time has elapsed (manager can be set optional).
+
+# Use-case off-chain activities
+
+We propose to extend the BitShares Blockchain in that each account is provided an additional freedom and possibilities 
+in configurating their vesting balances with locked up funds.
+
+## Exchange perspective
+
+### Customer sign up
+When a new customer signs up with the centralized, external exchange (CEX), a BitShares account must be uniquely linked. This can be done e.g. with a signed message. 
+
+### Customer on-boarding
+The centralized, external exchange (CEX) must monitor all vesting balances that are created on the BitShares Blockchain.
+If a managed vesting balance is recognized that has this CEX as settlement authority and otherwise matching 
+parameters (delay time period, cliff period), the asset and amount are credited to the customers account on the CEX.
+
+The customer could have the option to either 
+1. have the CEX create everything (BitShares account and vesting balances), CEX must provide the customer with the private keys in a safe manner
+2. create the BitShares account by itself, link it to the CEX account and let the CEX create the vesting balances
+3. create the BitShares account and vesting balance itself, signup at CEX and linking of accounts can also happen afterwards
+
+Once a vesting balance is created, the customer can charge it with a new operation introduced in specifications.
+
+### Customer off-boarding
```

Comment:
Please consider defining withdrawal and charging/depositing more explicitly to clarify that the accounts being described are both on BitShares.  Maybe something like

Withdrawal: from the "vesting balance" on BitShares to a user's "regular" account on BitShares
Charging: from a user's "regular" account on BitShares to his "vesting balance" on BitShares

Reply:
Refined